nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/test/java/org/nuxeo/ecm/platform/ec/notification/TestRenderingService.java

```diff
@@ -1,3 +1,21 @@
+/*
+ * (C) Copyright 2007-201999999999http://nuxeo.com/) and others.
```

Comment:
Typo :)

Reply:
it seems that I've discovered a new VIM command but I don't remember which one :/